dependency/vault_read_test.go

```diff
@@ -218,18 +227,8 @@ func TestVaultReadQuery_Fetch(t *testing.T) {
 func TestVaultReadQuery_Fetch_KVv2(t *testing.T) {
 	t.Parallel()
 
-	clients, vault := testVaultServer(t)
-	defer vault.Stop()
-
-	// Enable v2 kv for versioned secrets
-	vc := clients.Vault()
-	if err := vc.Sys().TuneMount("secret", api.MountConfigInput{
-		Options: map[string]string{
-			"version": "2",
-		},
-	}); err != nil {
-		t.Fatalf("Error tuning secrets engine: %s", err)
-	}
+	clients, vault := testVaultServer("read_fetch_v2", "2")
+	secrets_path := vault.secrets_path
```

Comment:
snake case -> camel case

Reply:
Changing the struct variable to camelCase... leaving local alone.